mobile/test/utils/extensions/kya_test.dart

```diff
@@ -21,6 +21,7 @@ void main() {
     activeTask: 1,
     status: KyaLessonStatus.todo,
     completionMessage: 'Lesson completed!',
+    hasCompleted: false,
   );
```

Comment:
Glad that you are actively including the unit tests! Keep it up @katendejericho5 !